Cargo.toml

```diff
@@ -21,6 +21,7 @@ security-framework-sys = "2.0.0"
 lazy_static = "1.4.0"
 libc = "0.2"
 tempfile = "3.1.0"
+rand = "0.8.4"
```

Comment:
Can you use https://docs.rs/security-framework/latest/security_framework/random/struct.SecRandom.html instead to avoid the extra dependency? You should be able to just hex encode the randomness or whatever to make a string.

Reply:
`tempfile` depends on `rand` already, so this is not a new dependency. But we can use `SecRandom` instead.